src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -932,6 +934,16 @@ public static class ParameterIdentification
          public static readonly string TimeProfileConfidenceIntervalAnalysis = "Confidence Interval";
          public static readonly string TimeProfileVPCIntervalAnalysis = "Visual Predictive Check Interval";
 
+         public static string DeviationLineNameUpper(float foldValue)
+         {
```

Comment:
Please use string concatenation
```
public static string DeviationLineNameUpper(float foldValue)=> $"{foldValue}-fold deviation Upper";
```
Same with the other line

Reply:
done